### ISSUE TITLE
Video doesn't load

### ISSUE THREAD

**Issue by KeyShr3kt** [kevinatown/MMM-Screencast#38]
I installed everything as you said. It showed me that there are 13 vulnerabilities, but I read that it doesn't affect the module, so I moved on. I installed the module, I connected the phone to the mirror, but when I wanted to play a video, it just loaaaaaaaaaaads and doesn't do anything else. Can you think for a solution? Thanks!

**Comment by KeyShr3kt** [comment]
I really like the idea of the module and I really want to make it work. By the way, I'm using a bluetooth speaker connected to the Pi and it seems that they are a little problematic. Maybe there's the problem.

**Comment by KeyShr3kt** [comment]
Sooo, I just tried to play via analog connection to the same speakers and it loads the video right when I switch to analog. It's a bluetooth problem. :( Do you have any idea what can it be?

**Comment by kevinatown** [comment]
Hey @KeyShr3kt! Sorry to hear MMM-Screencast is not working with your bluetooth speaker. I probably need to buy one and set it up to figure out what's going on here.
Do you think you can give me some more information on you hardware/configuration to see if there is anything that sticks out?

- what speaker are you using?
- did you do any configuration?
- what os/hardware is the MM running on? (Raspbian on a pi?)
- any other information you can provide will be super helpful. 

I'm kinda lost for what the issue would be but am hoping that it's probably something super silly like a configuration or something.

Thanks!

**Comment by Blackhawk50000** [comment]
I have the same problem. my youtube app says loading. but nothing happens on the mirror. 
i tryed it with analog 3,5mm headphonejack... 
nothing :-(

**Comment by kevinatown** [comment]
Hmm. I'm not quite sure whats up with using different sound outputs. I will try and find time to look into this. In the meantime can you try from the MMM-Screencast directory:
```bash
git pull
rm -rf node_modules
npm i
```

I'm hoping, but not guaranteeing that just updating electron will work.

**Comment by ed0zanki** [comment]
nope.. as soon as you use a bluetoothSpeaker (EasyAcc here) ist does not work (did update electron). remove the bluetoothSpeaker...everything is fine .. (at least the video runs :) )

**Comment by kevinatown** [comment]
Hmm ok, I wonder if it's something to do with YouTube/Chromium autoplay? If so I may have a fix here in a bit. I've been on vacation and are only getting back in the shake of things, so it might be a bit longer till I can get the MR out.
